src/components/CustomLink/CustomLink.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+
+type Props = {
+  name: string;
+  to: string;
+  sex: string;
+};
```

Comment:
You can pass the whole person
![image](https://github.com/mate-academy/react_people-table-basics/assets/63243136/89225097-0402-4948-82e1-f20bca50539e)